basic-sql-commands.md

```diff
@@ -194,16 +194,73 @@ INSERT INTO table1 (column_d, column_c, column_b, column_a) VALUES
     ('aaaa', DATE '2016-01-01', DEFAULT, 12345);
 ```
 ```
-+----------+----------+------------+----------+
-| column_a | column_b |  column_c  | column_d |
-+----------+----------+------------+----------+
-| 12345    |     NULL | 2016-01-01 | aaaa     |
-+----------+----------+------------+----------+
++----------+----------+---------------------------------+----------+
+| column_a | column_b |            column_c             | column_d |
++----------+----------+---------------------------------+----------+
+|    12345 |     NULL | 2016-01-01 00:00:00 +0000 +0000 | "aaaa"   |
++----------+----------+---------------------------------+----------+
 ```
 
 ## Query a Table
 
+To query a table, use the `SELECT` command followed by the columns to be returned and the table from which to retrieve the data:
```

Comment:
I was going off of the postgres docs, which use the term `command`: http://dev.mysql.com/doc/refman/5.7/en/sql-syntax-data-definition.html

Cassandra uses `command` as well for their query language.  

But I see that mysql, voltdb, and nuodb use `statement`: http://dev.mysql.com/doc/refman/5.7/en/sql-syntax-data-definition.html
https://docs.voltdb.com/UsingVoltDB/AppxDDL.php
http://doc.nuodb.com/display/doc/SQL+Reference+Pages

I'm happy switching to `statement`.